mappings/net/minecraft/world/biome/source/util/OverworldBiomeParameters.mapping

```diff
@@ -0,0 +1,122 @@
+CLASS net/minecraft/unmapped/C_gdyvdnno net/minecraft/world/biome/source/util/OverworldBiomeParameters
+	FIELD f_bqnzgain nonFrozenTemperature Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_dotshupm VALLEY_SIZE F
+	FIELD f_dphgjpnv hillBiomes [[Lnet/minecraft/unmapped/C_hcaelhhx;
+	FIELD f_egazlcyk PEAK_SIZE F
+	FIELD f_exvbssgp temperatures [Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_flvkuxxu fullRange Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_hacgctkf plateauBiomesVariant [[Lnet/minecraft/unmapped/C_hcaelhhx;
+	FIELD f_hkzwnxji coastContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_kgnoscjw nearInlandContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_ksqbybun erosions [Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_nipjbvyl mushroomFieldsContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_pwzmxluc regularBiomes [[Lnet/minecraft/unmapped/C_hcaelhhx;
+	FIELD f_qfzszkny deepOceanContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_rhpdqupa frozenTemperature Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_rvcyduuk plateauBiomes [[Lnet/minecraft/unmapped/C_hcaelhhx;
+	FIELD f_snbdsnqj oceanContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_ugfrbvdx oceanBiomes [[Lnet/minecraft/unmapped/C_hcaelhhx;
+	FIELD f_vbpnuepb farInlandContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_vurbgrlc PEAK_END F
+	FIELD f_wrwcwcob inlandContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_wzzxoteg midInlandContinentalness Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_xpcohlkz regularBiomesVariant [[Lnet/minecraft/unmapped/C_hcaelhhx;
+	FIELD f_ywphpjca humidities [Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;
+	FIELD f_zddmujpv PEAK_START F
+	METHOD m_agfvgbns pickBeachBiome (II)Lnet/minecraft/unmapped/C_hcaelhhx;
+		ARG 1 temperature
+		ARG 2 humidity
+	METHOD m_aluorree addBiomesTo (Ljava/util/function/Consumer;)V
+		ARG 1 biomeEntryConsumer
+	METHOD m_asupgdcz addUndergroundBiomesTo (Ljava/util/function/Consumer;)V
+		ARG 1 parameters
+	METHOD m_awfztaxz addMidSliceTo (Ljava/util/function/Consumer;Lnet/minecraft/unmapped/C_ygdawqho$C_mbbpocmp;)V
```

Comment:
I'd say you could name these like `addMidBiomesTo` instead of `addMidSliceTo`, so it's at least a bit clearer what they do.